### PR TITLE
Add Alexandria chainspec file

### PR DESCRIPTION
**Pull request by mnaamani**
Removing reference to rome.json in readme
Fix instructions for running integration tests
Remove old files from testnets/
Added new alexandria chainspec file

### CODE REVIEW

File: node/README.md [hunk 1]
```diff
@@ -49,15 +49,15 @@ this script will build and run a fresh new local development chain (purging exis
 Use the `--chain` argument, and specify the path to the genesis `chain.json` file for that public network. The JSON "chain spec" files for Joystream public networks can be found in [../testnets/](../testnets/).
 
 ```bash
-./target/release/joystream-node --chain testnets/rome.json
+./target/release/joystream-node --chain testnets/joy-tesnet-4.json
```
Comment (bwhm) [inline node/README.md]:
```suggestion
./target/release/joystream-node --chain testnets/joy-testnet-4.json
```

**Review by bwhm — CHANGES_REQUESTED** [review]
Not sure if if we can (or should) merge this before the `joy-testnet-4.json` is added?

**Comment by mnaamani** [comment]
okay lets wait to merge it until I add the chainspec to the repo, then will merge this into master

**Review by bwhm — APPROVED** [review]
LGTM!

**Pull request merged**